man/dockerd.8.md

```diff
@@ -84,7 +85,12 @@ following format.
 # OPTIONS
 
 **--add-runtime**=[]
-  Set additional OCI compatible runtime.
+  Set additional OCI compatible runtime. For example:
+
+    $ sudo dockerd --add-runtime runc=runc --add-runtime custom=/usr/local/bin/my-runc-replacement
+
+  Note that defining runtime arguments via the command line is not supported, you could define
+  arguments in daemon configuration file.
```

Comment:
Perhaps it would be good to use the same text as is used in the online reference, as it also shows how to use the `daemon.json` file for this (https://github.com/docker/docker/blob/master/docs/reference/commandline/dockerd.md#docker-runtime-execution-options)

```markdown
Additional runtimes can be registered with the daemon either via the
configuration file or using the **--add-runtime** command line argument.

The following is an example adding 2 runtimes via the configuration:

    {
      "default-runtime": "runc",
      "runtimes": {
        "runc": {
          "path": "runc"
        },
        "custom": {
          "path": "/usr/local/bin/my-runc-replacement",
          "runtimeArgs": [
            "--debug"
          ]
        }
      }
    }

This is the same example via the command line:

    $ sudo dockerd --add-runtime runc=runc --add-runtime custom=/usr/local/bin/my-runc-replacement

> **Note**: defining runtime arguments via the command line is not supported.
```